.github/workflows/csv-to-csvw.yml

```diff
@@ -52,52 +53,68 @@ jobs:
+                  file_secondary_extension="${file_secondary##*.}"
+                  if [[ "${file_secondary_name}.${file_secondary_extension}" == "${file_path}.json" ]]; then
+                    config_file="${file_secondary_name}.json"
+                  fi
+                done
+              elif [[ $file_extension == "json" ]]; then
+                config_file=${file}
+                if [[ -f "${file_path}/${file_name}.csv" ]]; then
+                  csv_file="${file_path}/${file_name}.csv"
                 fi
-              done
-              
-              if [[ -f $config_file ]]; then
-                echo "Config for ${csv_file} is available: ${config_file}"
-                echo "Building CSV-W"
-                csvcubed build $csv_file -c $config_file --out $out_path --validation-errors-to-file
-              else
-                echo "Config for ${csv_file} is not available"
-                echo "Building CSV-W"
-                csvcubed build $csv_file --out $out_path --validation-errors-to-file
               fi
               
-              echo "Inspecting CSV-W"
-              inspect_output_file="${out_path}inspect_output.txt"
-              if [[ -f $inspect_output_file ]]; then
-                  rm $inspect_output_file
-              fi
-
-              mapfile -d $'\0' inspectable_files < <(find "${GITHUB_WORKSPACE}/${out_path}" -name "*.csv-metadata.json" -type f -print0)
-              for file in "${inspectable_files[@]}"; do
-                echo "Inspecting file ${file}"
-                csvcubed inspect $file >> "${out_path}inspect_output.txt"
-              done
+              echo "csv_file for processing: ${csv_file}"
               
-
-              unset $config_file
-              echo "Finished processing file ${file}"
-              echo "::set-output name=has_outputs::${{ toJSON(true) }}"
+              if [[ -f $csv_file ]]; then
+                if [[ -f $config_file ]]; then
+                  echo "Config for ${csv_file} is available: ${config_file}"
+                  echo "Building CSV-W"
+                  csvcubed build $csv_file -c $config_file --out $out_path --validation-errors-to-file
+                else
+                  echo "Config for ${csv_file} is not available"
+                  echo "Building CSV-W"
+                  csvcubed build $csv_file --out $out_path --validation-errors-to-file
+                fi
+                
+                echo "Inspecting CSV-W"
+                mapfile -d $'\0' inspectable_files < <(find "${GITHUB_WORKSPACE}/${out_path}" -name "*.csv-metadata.json" -type f -print0)
+               
+                for file in "${inspectable_files[@]}"; do
+                  echo "Inspecting file ${file}"
+                  file_path="${file%.*}"
+                  file_name="${file_path##*/}"
+                  inspect_output_file="${out_path}${file_name}_inspect_output.txt"
+
+                  if [[ -f $inspect_output_file ]]; then
```

Comment:
You could just replace this if statement and the following line with `csvcubed inspect "$file" > "$inspect_output_file"` that will overwrite the contents without appending every time.